tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -288,7 +288,7 @@ object TastyFormat {
    *  compatibility, but remains backwards compatible, with all
    *  preceeding `MinorVersion`.
    */
-  final val MinorVersion: Int = 3
+  final val MinorVersion: Int = 2
```

Comment:
these tasty changes have to be reverted

Reply:
at least I think they do, the next release is meant to be 3.3.0?

Reply:
Yes, absolutely. I should have spotted that earlier.